README.md

```diff
@@ -37,7 +37,7 @@ A `Promise` MUST be in one of three states: `pending`, `succeeded`, `failed`.
 
 ## Consumption
 
-A `Promise` MUST implement `Interop\Async\Promise` and thus provide a `when()` method to access its current or eventual value or reason.
+A `Promise` MUST implement `Interop\Async\Promise` and thus provide a `when()` method to access its value or reason.
```

Comment:
I'd retain the "current or future" here as there is no other temporal indication here, currently suggesting that the `current` value/reason could be meant.